src/components/common/ParentNavTopBar.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Typography, Menu, Dropdown, Button } from 'antd';
 import { Link, useHistory } from 'react-router-dom';
-import { useOktaAuth } from '@okta/okta-react';
+import { useAuth0 } from '@auth0/auth0-react'; //replaces okta
```

Comment:
While I appreciate the comment, I don't believe it will need to stay in the codebase as Okta will not be around anymore. From the PR it's fairly self-explanatory as you see both the removed line and the added line with both saying they are an Auth tool.